build/win32/i18n/messages.ru.isl

```diff
@@ -1,8 +1,9 @@
 [CustomMessages]
-AddContextMenuFiles=�������� �������� "������� � ������� %1" � ����������� ���� ����� ���������� Windows
-AddContextMenuFolders=�������� �������� "������� � ������� %1" � ����������� ���� �������� ����������
-AssociateWithFiles=���������������� %1 � �������� ��������� ��� �������������� ����� ������
-AddToPath=�������� � PATH (�������� ����� ������������)
-RunAfter=��������� %1 ����� ���������
-Other=������:
-SourceFile=�������� ���� %1
\ No newline at end of file
+AddContextMenuFiles=�������� �������� "������� � ������� %1" � ����������� ���� ����� ���������� Windows
```

Comment:
Lastly i want to say. It seems your editor changes charsets of files. May this breaks

Reply:
Sorry for one by one comment 😕

Reply:
I will look into it later today

Reply:
I am using VS Code to do the changes, but perhaps the encodings are not ok on my PC